modules/adot/templates/adot-values.yaml.tpl

```diff
@@ -136,6 +114,29 @@ adotCollector:
           limit_mib: 100
           check_interval: 5s
       exporters:
+        awsemf/prometheus:
+          dimension_rollup_option: NoDimensionRollup
+          log_group_name: adot-log-group-dasmeta
```

Comment:
I think we should use this assignment
`log_group_name: "${log_group_name}"`
instead of hardcoding the log group name.

Reply:
maybe better have this optional, so that if not passed no logs send for metrics. We actually need this for debugging only and having this all the time enabled can impact on costs